### PR TITLE
Next fixes

### PR DESCRIPTION
**Pull request by macumber**

### CODE REVIEW

File: lib/psi.rb [hunk 1]
```diff
@@ -1850,14 +1850,12 @@ def processTBD(
     surface[:type] = :wall if typ.include?("wall")
 
     unless s.construction.empty?
-      construction = s.construction.get.to_Construction.get
+      construction = s.construction.get.to_ConstructionBase.get
```
Comment (macumber) [inline lib/psi.rb:1853]: The issue here was that construction was an `OS:Construction:FfactorGroundFloor` object and not an `OS:Construction`, `PlanarSurface::construction` returns a `ConstructionBase`.  `ConstructionBase` doesn't have a `layers` method, I figured you didn't really need to check if `index < construction.layers.size` as long as `deratableLayer` wouldn't return an invalid layer (which it looks like it won't).

Comment (macumber) [reply lib/psi.rb:1853]: @rd2 FYI

**Comment by macumber** [comment]
This fixes the two test suites so they run for me.  However, it seems like the energy results are the same for each TBD option set, is that expected?

**Comment by macumber** [comment]
I'm also seeing a lot of errors with `bundle exec rake spec`

File: lib/psi.rb (hunk 1 above)
Comment (rd2) [reply lib/psi.rb:1853]: @macumber: Thanks. Odd that this never popped up in previous testing.

**Pull request merged**